src/platform/packages/private/kbn-grid-layout/grid/use_grid_layout_events/sensors/keyboard/keyboard.ts

```diff
@@ -29,70 +29,67 @@ const preventDefault = (e: Event) => e.preventDefault();
 const disableScroll = () => window.addEventListener('wheel', preventDefault, { passive: false });
 const enableScroll = () => window.removeEventListener('wheel', preventDefault);
 
-const scrollToActiveElement = (shouldScrollToEnd: boolean) => {
```

Comment:
Not sure if something got removed here, but atm the resize keyboard behavior doesn't properly scroll when the resize handle is outside of the screen:

<img width="861" alt="Screenshot 2025-05-15 at 23 03 28" src="https://github.com/user-attachments/assets/b62d4971-93d5-4993-bda5-bff48be2cd8b" />

Reply:
I'm not seeing this:


https://github.com/user-attachments/assets/3a62c3ab-86f0-4ec0-bddb-efd9b45dee0b

Do you have some more specific repro steps? 🙇

Reply:
You fixed it here, just checked 🎉 
520d5399f0ef0ae07e2f3788e26535a951b2be2b